src/advanced_inv.cpp

```diff
@@ -1324,6 +1310,27 @@ bool advanced_inventory::action_move_item( advanced_inv_listitem *sitem,
         // exit so that the activity can be carried out
         exit = true;
 
+    } else if (to_container) {
+        int remaining_amount = amount_to_move;
+        for( item *itm : sitem->items ) {
+            if( remaining_amount <= 0 ) {
+                break;
+            }
+            item &put = *itm;
+            item *container = dpane.get_container_location().get_item();
+            if ( !container->can_contain( put ) ) {
+                popup( _( "Cannot place any more into target container!") );
+                return false;
+            }
+            //TODO Subtract turns from player
```

Comment:
I was planning to get the turns to remove/add from the container here and manually add/remove them from the player. Is that a sensible solution here or is there a better way to achieve this?